katcp/resource.py

```diff
@@ -894,9 +892,7 @@ def handle_update(sensor, reading):
             # This handler is called whenever a sensor update is received
             try:
                 assert sensor is self
-                cond_matched = condition_test(reading)
-                status_matched = reading.status == status or status is None
```

Comment:
This explicit status check was originally included by @brickZA ... would like to hear his thoughts about dropping it.